spec/services/out_of_band_session_accessor_spec.rb

```diff
@@ -13,17 +13,17 @@
 
   describe '#ttl' do
     it 'returns the remaining time-to-live of the session data in redis' do
-      store.put_pii({ first_name: 'Fakey' }, 5.minutes.to_i)
+      store.put_pii(123, { first_name: 'Fakey' }, 5.minutes.to_i)
```

Comment:
Would it make sense to `let(profile_id) { 123 }` for clarity?

Reply:
Hmm, I can do something like that.